nix/process-compose/default.nix

```diff
@@ -37,21 +37,22 @@ in
 
   config.outputs =
     let
-      mkProcessComposeWrapper = { name, tui, httpServer, configFile, preHook, postHook }:
+      mkProcessComposeWrapper = { name, arguments, preHook, postHook, }:
         pkgs.writeShellApplication {
           inherit name;
           runtimeInputs = [ config.package ];
           text = ''
-            export PC_CONFIG_FILES=${configFile}
-            ${
-              # Once the following issue is fixed we should be able to simply do:
-              # export PC_DISABLE_TUI=${builtins.toJSON (!config.tui)}
-              # https://github.com/F1bonacc1/process-compose/issues/75
-              if tui then "" else "export PC_DISABLE_TUI=true"
-            }
             ${preHook}
 
-            set -x; process-compose ${httpServer.outputs.cliOpts} "$@"; set +x
+            params=(${arguments.global})
+            set +u
+            if [ -z "$1" ] || [[ "$1" == "up" ]] || [[ "$1" == -* ]] ; then
```

Comment:
If we're not running "up", don't pass all parameters. process-compose would not like it.

Reply:
I think this should be addressed in `main` branch. If not, let me know.